datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -444,6 +461,12 @@ def send_request(self, endpoint, scraper_config, headers=None):
             headers['accept-encoding'] = 'gzip'
         headers.update(scraper_config['extra_headers'])
 
+        # Add the bearer token to headers
+        bearer_token = scraper_config['_bearer_token']
+        if bearer_token is not None:
+            auth_header = {'Authorization': 'Bearer {}'.format(bearer_token)}
+            headers.update(auth_header)
```

Comment:
did you check the behaviour of `headers.update()` in case of `Authorization: Bearer ...` is already present from `extra_headers` ?

Reply:
it will overwrite the field, don't you think that `bearer_token_auth` must be prioritized over `extra_headers`  to set `Authorization` ? we can still log and warning if  we're overwriting `Authorization`, wdyt?

Reply:
overwrite is fine for me.

I'm just wondering if someone set `bearer_token_auth = true` but also provide the `Authorization` header. Maybe you juste need to add a bit more info in the conf.yaml.example  saying that `Authorization` header from `extra_headers` will be replace in case of `bearer_token_auth = true`

Reply:
will do that, thanks 👍